templates/etc/apache2/sites-available/default.conf.j2

```diff
@@ -12,7 +12,8 @@
 
 {% if item.redirect_http|d() %}
     {{ apache__tpl_macros.get_redirect( item.redirect_http_code|d(307), "/", item.redirect_http) | indent(4) }}
-{% elif item.redirect_to_https|d(True) | bool %}
+{% elif item.redirect_to_https|default(apache__redirect_to_https) | bool %}
```

Comment:
`d` for `default` is quite common in DebOps since we use the default filter all day long. Can you change that back?